src/ssmDocument/util/validateDocumentName.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { localize } from '../../shared/utilities/vsCodeUtils'
+
+export function validateDocumentName(name: string): string | undefined {
```

Comment:
Why put this in its own file? Could live in `util/util.ts`. 

Also what's in `util/util.ts` currently (`openAndSaveDocument()`) looks like it could live somewhere in [shared/utilities](https://github.com/aws/aws-toolkit-vscode/tree/master/src/shared/utilities).